### ISSUE TITLE
+1 for an official dart/websocket client implementation 

### ISSUE THREAD

**Issue by aktxyz** [nats-io/nats-server#2852]

**Comment by kozlovic** [comment]
@aktxyz We already have some websocket clients: 
- https://github.com/nats-io/nats.ws
- https://github.com/nats-io/nats.go (just use `ws://` or `wss://` URL scheme)

**Comment by aktxyz** [comment]
> @aktxyz We already have some websocket clients:
> 
> * https://github.com/nats-io/nats.ws
> * https://github.com/nats-io/nats.go (just use `ws://` or `wss://` URL scheme)

Yep, I saw those.  I was looking for a good dart client (to use in a flutter app).  There are several dart pub dev (https://pub.dev/), but a bit incomplete and buggy.
